src/content/docs/dev/send-tokens/sui/intro.mdx

```diff
@@ -47,19 +85,70 @@ Once triggered, ITS will generate a unique [token id](#token-id) from the coin's
 
 See [here](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/sui/its-example.js#L195) for an example of registering a new coin.
 
-### Deploy Remote Interchain Token
+## Register Coin Metadata
+
+The `register_coin_from_metadata()` function is used to register the [TokenMetadata](https://github.com/axelarnetwork/axelar-amplifier/blob/main/packages/interchain-token-service-std/src/primitives.rs#L82) msg type with [ITS Hub](/dev/amplifier/its-hub/introduction/) in order to register token data in ITS hub.. This function is required to be called on each chain where you token is being integrated to ITS on if you are planning on linking a custom token. 
+
+Note: This function is separate from [register_coin_from_metadata](#register-coin-from-metadata), which is used for registering canonical tokens for Sui.
+
+The `register_coin_from_metadata()` returns a [MessageTicket](/dev/general-message-passing/sui/gmp-contracts/#message-ticket) and takes the following two parameters:
+
+1. `its`: The ITS object that will be updated once the token's metadata is registered.
+1. `coin_metadata`: The metadata of the coin to be registered.
+
+It is written as follows:
+
+```rust
+public fun register_coin_metadata<T>(self: &InterchainTokenService, coin_metadata: &CoinMetadata<T>): MessageTicket {}
+```
+
+
+
+## Link Coin
+
+If you are looking to connect your custom coin on Sui to a custom coin on another blockchain, such as Ethereum, you will need to *link* the coin between the two chains. The linking process can only take place after the [register_custom_coin](#register-custom-coin) function has been called. The `link_coin()` function is defined as follows:
+
+The function takes seven parameters and returns a [MessageTicket](/dev/general-message-passing/sui/gmp-contracts/#message-ticket). 
+
+1. `its`: The ITS object that will be updated once the new token is linked.
+1. `deployer` The the deployer’s [channel](/dev/general-message-passing/sui/gmp-contracts/#channel) capability. Used to derive the custom [tokenId](#token-id) with the `salt` and `chain_name`, so only that channel’s holder can link that custom token.
+1. `destination_chain`: The name of the chain where the token is being linked to.
+1. `destination_token_address`: The address of the token the sui coin is being linked to.
+1. `token_manager_type`: The [type of manager](#coin-management) to be deployed on the destination chain.
+1. `link_params` A 32 byte representation of an address to be used as the operator on the destination chain.
```

Comment:
> ~~1. `link_params` A 32 byte representation of an address to be used as the operator on the destination chain.~~

1. `link_params` An array of bytes representing an address to be used as the operator on the destination chain.